seatunnel-dist/release-docs/LICENSE

```diff
@@ -937,13 +819,34 @@ The text of each license is also included at licenses/LICENSE-[project].txt.
      (Mozilla Public License Version 2.0) jamon-runtime (org.jamon:jamon-runtime:2.4.1 - http://www.jamon.org/jamon-java-parent/jamon-runtime/)
      (Mozilla Public License, Version 2.0) Mozilla Rhino (org.mozilla:rhino:1.7.11 - https://developer.mozilla.org/en/Rhino)
 
+
 ========================================================================
 Unicode License
 ========================================================================
 
 The following components are provided under the Unicode License. See project link for details.
 The text of each license is also included at licenses/LICENSE-[project].txt.
 
-     (Unicode/ICU License) ICU4J (com.ibm.icu:icu4j:60.2 - http://icu-project.org/)
      (Unicode/ICU License) ICU4J Charset Provider (com.ibm.icu:icu4j-charset:60.2 - http://icu-project.org/)
      (Unicode/ICU License) ICU4J Locale Service Provider (com.ibm.icu:icu4j-localespi:60.2 - http://icu-project.org/)
+
+
+========================================================================
+Other License
+========================================================================
+
+The following components are provided under some minority Licenses. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+
+     (Unknown license) Antlr 3.4 Runtime (org.antlr:antlr-runtime:3.4 - http://www.antlr.org)
+     (Unknown license) commons-beanutils (commons-beanutils:commons-beanutils:1.7.0 - no url defined)
+     (Unknown license) commons-pool (commons-pool:commons-pool:1.5.4 - no url defined)
+     (Unknown license) Java Transaction API (javax.transaction:jta:1.1 - http://java.sun.com/products/jta)
+     (Unknown license) jsp-api (javax.servlet.jsp:jsp-api:2.1 - no url defined)
+     (APL2) msgpack-core (org.msgpack:msgpack-core:0.9.0 - https://msgpack.org/)
+     (Unknown license) oro (oro:oro:2.0.8 - no url defined)
+     (The Go license) re2j (com.google.re2j:re2j:1.1 - http://github.com/google/re2j)
+     (Unknown license) servlet-api (javax.servlet:servlet-api:2.5 - no url defined)
+     (Unknown license) sigar (org.hyperic:sigar:1.6.5.132 - no url defined)
```

Comment:
These should be recognized by human beings, `Other License` is a potential license risk to me, as they are `Unknown`

Reply:
Hi @kezhenxu94, thanks for your comment. Yes, the `license-maven-plugin` plugin can exclude unknown license, but I think we shouldn't ignore them, we should clean up our dependencies tree to avoid unknown licenses

Reply:
> Hi @kezhenxu94, thanks for your comment. Yes, the `license-maven-plugin` plugin can exclude unknown license, but I think we shouldn't ignore them, we should clean up our dependencies tree to avoid unknown licenses

I don't mean we should ignore them, instead, we should identify them manually, or remove them if they don't have a valid license

Reply:
Hi @kezhenxu94, the latest version script already supports the handling of unknown licneses, and all existing unknown protocols have found the corresponding licenses and are correctly marked